x/checkpoint/types/msg.go

```diff
@@ -132,7 +132,6 @@ func (msg MsgCheckpointAck) Route() string {
 
 // GetSigners returns signers
 func (msg MsgCheckpointAck) GetSigners() []sdk.AccAddress {
-	//return []sdk.AccAddress{sdk.AccAddress([]byte(msg.From))}
 	addr, _ := sdk.AccAddressFromHex(msg.From)
```

Comment:
Please add error handler

Reply:
I think we can't return error here because it is returning sdk.AccAddress , instead of throwing error we can return empty sdk.AccAddress 
```
func (msg MsgCheckpointNoAck) GetSigners() []sdk.AccAddress {
	addr, err := sdk.AccAddressFromHex(msg.From)
	if err != nil {
		return []sdk.AccAddress{}
	}
	return []sdk.AccAddress{addr}
}
```